### DISCUSSION TITLE
Can't get PigmentCSS to work within Storybook's preview

### DISCUSSION THREAD

**Discussion by axeleroy** [storybookjs/storybook#26768] [Help]
### Summary

I set up a Next.js project with [PigmentCSS](https://github.com/mui/material-ui/tree/next/packages/pigment-css-react) and tried to set up Storybook on top but styling is not applied within Storybook's preview.

Adding `import "@pigment-css/react/styles.css";` in `preview.ts` unfortunately doesn't work :/

### Create a reproduction

https://stackblitz.com/edit/github-pa8f6a-6ujgan

**Comment by austinbiggs** [comment]
++ to this: I spent a few minutes with your repro and couldn't get the Pigment CSS to load. I'll keep an eye on this topic and continue noodling on a solution here, as I'm hoping to migrate to Pigment as well. Storybook support is the only hitch I'm currently facing.

**Comment by paulm17** [comment]
This is my last blocker as well.  I can get the vite storybook to work, but then it decides to optimise the dependencies and then it cascades with more issues.   Thanks vite 🙄 

```
15:42:11 [vite] Pre-transform error: Internal server error: Soft-invalidated module "/@fs/Users/paul/development/src/github/raikou/packages/@raikou/core/esm/components/Combobox/ComboboxDropdown/ComboboxDropdown.mjs" should not have existing transform result
```

I can't even get pigment-css to load (with nextjs).  I get the "you are trying to use css, etc... error".  When I have `withPigment` in the next config.

What's worse, is that it works perfectly fine in the default next project, just not for storybook.

**Comment by hoektoe** [comment]
Same issue, cannot get my styling to come through to storybook
